corehq/apps/hqwebapp/static/hqwebapp/js/select_2_ajax_widget.js

```diff
@@ -1,20 +1,21 @@
-{% load hq_shared_tags %}
-<script>
+hqDefine("hqwebapp/js/select_2_ajax_widget", function() {
     $(function(){
-        $('#{{ id }}').select2({
-                multiple: {{ multiple|JSON }},
+        $(".hqwebapp-select2-ajax").each(function() {
+            var data = $(this).data();
```

Comment:
`$(this)` is used a few times, which might be wroth reducing. it's also the same as doing `$(arguments[1])`